stable/rabbitmq/templates/certs.yaml

```diff
@@ -0,0 +1,19 @@
+{{ if and (not .Values.rabbitmq.tls.existingSecret) ( .Values.rabbitmq.tls.enabled) }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "rabbitmq.fullname" . }}-certs
+  labels:
+    app: {{ template "rabbitmq.name" . }}
+    chart: {{ template "rabbitmq.chart" .  }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  ca_certificate.pem:
+    {{ .Values.rabbitmq.tls.caCertificate | b64enc | quote }}
+  server_certificate.pem:
+    {{ .Values.rabbitmq.tls.serverCertificate| b64enc | quote }}
+  server_key.pem:
+    {{ .Values.rabbitmq.tls.serverKey | b64enc | quote }}
```

Comment:
These values are required when `rabbitmq.tls.enabled` is `true`, right? In that case, I'd use the approach below:
```
required "A valid .Values.rabbitmq.tls.serverKey entry required!" .Values.rabbitmq.tls.serverKey | b64enc | quote }}
```

Reply:
I have added the `required` for the three certs files.